.github/workflows/buildTestWindows.yml

```
@@ -0,0 +1,26 @@
+name: Java CI
+
+on: [push]
```

Let's make this run for all PRs (including this one!), not just AFTER I merge, so [like here] (https://github.com/vorburger/MariaDB4j/blob/main/.github/workflows/codeql-analysis.yml), use:

```suggestion
on:
  push:
    branches: [ main ]
  pull_request:
    # The branches below must be a subset of the branches above
    branches: [ main ]
```